client/src/templates/Challenges/components/side-panel.css

```diff
@@ -8,7 +8,11 @@
 
 .instructions-panel {
   padding: 0 10px;
-  height: 100%;
-  width: 100%;
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
```

Comment:
Altering the styles for the .instructions-panel does not seem to have an effect.

Reply:
In Chrome, Edge, and Brave PC browsers, Actually, when we do not use this 5-line style, in the mobile layout and in the Challenges (not Steps like new RWD), the child elements of `.instructions-panel` overflow from the `.instructions-panel` due to the amount of their own content, making the `body` element's vertical scrollbar appear. 

I used them to prevent the bar from appearing.

Reply:
Thanks. now the redundant `display: flex` was removed from the code so no longer need to use `position: absolute`.